Description: Fixup vcversion cruft
 vcversion looks stupid to me. Putting a semver version in version.txt just
 didn't work for me. So I'm being even more stupid and hardcoding the version
 in setup.py. If someone knows a better way to fix it, let me know.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2015-10-09

Index: python-jsonschema/jsonschema/_version.py
===================================================================
--- /dev/null
+++ python-jsonschema/jsonschema/_version.py
@@ -0,0 +1,5 @@
+
+# This file is automatically generated by setup.py.
+__version__ = '2.6.0'
+__sha__ = '1\n'
+__revision__ = '1\n'
Index: python-jsonschema/setup.py
===================================================================
--- python-jsonschema.orig/setup.py
+++ python-jsonschema/setup.py
@@ -27,6 +27,7 @@ extras_require = {
 
 setup(
     name="jsonschema",
+    version="2.6.0",
     packages=["jsonschema", "jsonschema.tests"],
     package_data={"jsonschema": ["schemas/*.json"]},
     setup_requires=["vcversioner>=2.16.0.0"],
@@ -39,5 +40,4 @@ setup(
     long_description=long_description,
     url="http://github.com/Julian/jsonschema",
     entry_points={"console_scripts": ["jsonschema = jsonschema.cli:main"]},
-    vcversioner={"version_module_paths" : ["jsonschema/_version.py"]},
 )
